includes/discovery/sensors/temperature/exos.inc.php

```diff
@@ -1,12 +1,35 @@
 <?php
 
-echo 'Exos ';
+/**
+ * exos.inc.php
+ *
+ * @author     Peca Nesovanovic <peca.nesovanovic@sattrakt.com>
+ */
+$oids = SnmpQuery::cache()->options('-OteQUsab')->walk('FCMGMT-MIB::connUnitSensorTable')->valuesByIndex();
```

Comment:
options should not be used with SnmpQuery

Reply:
done